zha/application/platforms/light/const.py

```diff
@@ -91,6 +91,7 @@ class ColorMode(StrEnum):
 EFFECT_COLORLOOP: Final[str] = "colorloop"
 EFFECT_RANDOM: Final[str] = "random"
 EFFECT_WHITE: Final[str] = "white"
+EFFECT_NONE: Final[str] = "None"
```

Comment:
Shouldn't this be `EFFECT_OFF = "off"` (not `EFFECT_NONE`)?
At least, that's what HA uses: [home-assistant/core/blob/homeassistant/components/light/__init__.py#L228-L233](https://github.com/home-assistant/core/blob/7820bcf2186c9e9d815c91c724e5187809b64d4b/homeassistant/components/light/__init__.py#L228-L233)

Reply:
Yep not sure wth I was looming at 😂. Good catch